docs/template-vm.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: linux-vm-template
+  annotations:
+    description: "OCP kubevirt linux, template"
+    tags: "kubevirt,ocp,template,linux"
+    miq.github.io/kubevirt-is-vm-template: true
+objects:
+- apiVersion: kubevirt.io/v1alpha1
+  kind: OfflineVirtualMachine
+  metadata:
+    name: ${NAME}
+  spec:
+    template:
+      spec:
+        domain:
+          cpu:
+            cores: ${CPU_CORES}
+          resources:
+            requests:
+              memory: ${MEMORY}
+          machine:
+            type: q35
+          devices:
+            interface:
```

Comment:
We do not yet allow setting any interfaces. Right we add one by default which  is not visible to the user. That will for sure change in the furute, but it will no have effect now.

Reply:
OK, will remove it